kiele

```diff
@@ -47,6 +48,10 @@ run_assemble() {
     iele-assemble "$run_file" "$@"
 }
 
+run_pyiele() {
+    python3 -m pyiele "$@"
+}
```

Comment:
```suggestion
run_pyiele() {
    python3 -m pyiele --host "${host}" --port "${port}" --output "${output_path}" "$@"
}
```

Reply:
This will have to be
```
python3 -m pyiele "$@" --host "${host}" --port "${port}" --output "${output_path}"
```
Because the command `{blackbox,test,compile,coverage}` is a positional parameter.

Reply:
On second look, this might not work at all. None of these parameters are in each of the commands.

Reply:
I was able to set the `run_pyiele()` function to Everett's suggestion by moving the `port`,`host`, and `output` parameters to the main parser (initially, they were subparser parameters).